src/main/java/mat/vsac/VsacService.java

```diff
@@ -699,4 +570,15 @@ private HttpEntity<MultiValueMap<String,String>> buildEntityWithUrlEncodedBody(M
         headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
         return new HttpEntity<>(body, headers);
     }
+    
+    private HttpEntity<String> getHeaderEntityWithAuthentication(String apiKey) {
+    	HttpHeaders headers = new HttpHeaders();
+      headers.add("Authorization", getBasicAuthenticationHeader("apikey", apiKey));
```

Comment:
Spring has a shortcut method for setting Basic Auth:
```suggestion
      headers.setBasicAuth("apikey", apiKey);
```